clojure_action.py

```diff
@@ -272,12 +268,12 @@ def _setOutput(name:str, value:str):
         logging.error(f"Error parsing licenses!")
     try:
         for vuln in test_res['vulnerabilities']:
-            vulns.add(VulnReport(vuln))
+            if(vulns['id'].startswith('snyk:lic:') is False):
```

Comment:
gonna be nitpicky one more time, sorry:

```python
if not vulns['id'].startswith('snyk:lic:'):
```

Reply:
Its good to be nitpicky with this stuff! Thanks for the feedback! I will reflect the change.

Reply:
Changed to: 
`if not vuln['id'].startswith('snyk:lic:'):`
Back to the correct variable.